Framework/include/QualityControl/Checker.h

```diff
@@ -1,31 +1,25 @@
 ///
 /// \file   Checker.h
```

Comment:
There is a little bit of inconsistency with naming of new `Checker` and `TaskDataProcessor`. They both share `framework::Task` interface and have associated factory, but have differerent naming convention. Maybe we should stick either to `Checker`/`Task`/`HistoMerger` or accordingly `*DataProcessor`.

Reply:
Aah, indeed. It does not inherit from framework::Task, though it has similar interface. And indeed, I wanted to have some common structure of these DPL devices. Then we could have one factory for them all.
I am fine with leaving it like it is, but when we discuss, we can consider also `TaskRunner`.

Reply:
Funny you mention `TaskRunner`, that also crossed my mind. We got a winner :)